py/ai/h2o/sparkling/Initializer.py

```diff
@@ -106,11 +111,12 @@ def check_different_h2o():
             command_sys_path = "import sys; sys.path = " + str(path_without_sw).replace("'", "\"") + ";"
             command_import_h2o = "import h2o; print(h2o.__version__)"
             full_command = "python -c '" + command_sys_path + command_import_h2o + "'"
-            previous_version = subprocess.check_output(full_command, shell=True, stderr=DEVNULL).decode('utf-8').replace("\n", "")
+            previous_version = subprocess.check_output(full_command, shell=True, stderr=DEVNULL).decode(
```

Comment:
nit: Can  we format the piece of code in a better way? Maybe:
```
previous_version = subprocess
    .check_output(full_command, shell=True, stderr=DEVNULL)
    .decode('utf-8')
    .replace("\n", "")
```

Reply:
👍